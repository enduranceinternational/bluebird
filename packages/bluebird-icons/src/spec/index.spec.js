import * as icons from '../index.js';

describe('@eigi/bluebird-icons', () => {
  it('matches snapshot', () => {
    expect(icons).toMatchSnapshot();
  });
});
