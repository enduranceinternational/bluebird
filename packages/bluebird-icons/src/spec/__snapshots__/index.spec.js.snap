// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@eigi/bluebird-icons matches snapshot 1`] = `
Object {
  "BarsRegular": [Function],
  "BellRegular": [Function],
  "BlueSkyDifference": [Function],
  "BlueSkyDifferenceSubtract": [Function],
  "BlueSkyDifferenceUnion": [Function],
  "BoxFullRegular": [Function],
  "BrowserOff": [Function],
  "BrowserOffline": [Function],
  "BrowserOnline": [Function],
  "BrowserRegular": [Function],
  "BrowserWarning": [Function],
  "BuildingRegular": [Function],
  "BuildingRegularOff": [Function],
  "CalendarDayRegular": [Function],
  "CheckCircleRegular": [Function],
  "CheckRegular": [Function],
  "CheckboxDisabled": [Function],
  "CheckboxHover": [Function],
  "CheckboxNotSelected": [Function],
  "CheckboxSelected": [Function],
  "ChevronDownRegular": [Function],
  "ChevronLeftRegular": [Function],
  "ChevronRightRegular": [Function],
  "ChevronUpRegular": [Function],
  "Close": [Function],
  "CommentAltRegular": [Function],
  "CommentsAltSolid": [Function],
  "CompassRegular": [Function],
  "CreditCardRegular": [Function],
  "EditRegular": [Function],
  "EllipsisVRegular": [Function],
  "ExclamationCircleRegular": [Function],
  "ExclamationTriangleRegular": [Function],
  "Expired": [Function],
  "ExternalLinkSquareAltRegular": [Function],
  "EyeRegular": [Function],
  "EyeSlashRegular": [Function],
  "Filter": [Function],
  "HomeLgRegular": [Function],
  "IdCardAltRegular": [Function],
  "InfoCircleRegular": [Function],
  "LineChart": [Function],
  "LockOff": [Function],
  "LockOpenRegular": [Function],
  "LockRegular": [Function],
  "LockSolid": [Function],
  "LongArrowLeftRegular": [Function],
  "LongArrowRightRegular": [Function],
  "MailBulkRegular": [Function],
  "MinusCircleRegular": [Function],
  "MobileAndroidAltSolid": [Function],
  "PaperAirplane": [Function],
  "PlusCircleRegular": [Function],
  "PlusRegular": [Function],
  "QrcodeLight": [Function],
  "QuestionCircleRegular": [Function],
  "RadioButtonDisabled": [Function],
  "RadioButtonNotSelected": [Function],
  "RadioButtonNotSelectedHover": [Function],
  "RadioButtonSelected": [Function],
  "SearchRegular": [Function],
  "ShieldCheckRegular": [Function],
  "ShoppingBagRegular": [Function],
  "ShoppingCartRegular": [Function],
  "SignOutRegular": [Function],
  "SlidersHRegular": [Function],
  "SortAmountDownRegular": [Function],
  "SortAmountUpRegular": [Function],
  "SortDownSolid": [Function],
  "SortUpSolid": [Function],
  "SpinnerThirdRegular": [Function],
  "Star": [Function],
  "TextField": [Function],
  "TimesCircle": [Function],
  "TimesLight": [Function],
  "TrashRegular": [Function],
  "UndoRegular": [Function],
  "UserAltSolid": [Function],
  "UserAstronautLight": [Function],
  "UserAstronautSolid": [Function],
  "UserCircleSolid": [Function],
  "WordpressSimpleBrands": [Function],
  "__esModule": true,
}
`;
